src/displayapp/screens/Music.cpp

```diff
@@ -248,25 +272,59 @@ void Music::OnObjectEvent(lv_obj_t* obj, lv_event_t event) {
   }
 }
 
-bool Music::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
-  switch (event) {
-    case TouchEvents::SwipeUp: {
+void Music::UpdateButtons() {
+  switch(currentButton)
+  {
+    case BTN_NEXT_PREV:
+      lv_obj_set_hidden(btnForward, true);
+      lv_obj_set_hidden(btnRewind, true);
+      lv_obj_set_hidden(btnVolDown, true);
+      lv_obj_set_hidden(btnVolUp, true);
+      lv_obj_set_hidden(btnNext, false);
+      lv_obj_set_hidden(btnPrev, false);
+      break;
+    case BTN_VOL:
+      lv_obj_set_hidden(btnForward, true);
+      lv_obj_set_hidden(btnRewind, true);
       lv_obj_set_hidden(btnVolDown, false);
       lv_obj_set_hidden(btnVolUp, false);
-
       lv_obj_set_hidden(btnNext, true);
       lv_obj_set_hidden(btnPrev, true);
+      break;
+    case BTN_FOR_REWIND:
+      lv_obj_set_hidden(btnForward, false);
+      lv_obj_set_hidden(btnRewind, false);
+      lv_obj_set_hidden(btnVolDown, true);
+      lv_obj_set_hidden(btnVolUp, true);
+      lv_obj_set_hidden(btnNext, true);
+      lv_obj_set_hidden(btnPrev, true);
+      break;
+    default:
```

Comment:
I don't know if this actually improves the code, but you could do something like
```cpp
  case BTN_NEXT_PREV:
  default:
    lv_obj_ ...
```

This would reduce code duplication and make clear what the default is.